input/resources/operationdefinition-knowledgeartifact-release.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<OperationDefinition xmlns="http://hl7.org/fhir">
+    <id value="KnowledgeArtifact-release"/>
+    <url value="http://hl7.org/fhir/uv/cmi/OperationDefinition/KnowledgeArtifact-release"/>
+    <version value="3.0.0"/>
+    <name value="KnowledgeArtifactRelease"/>
+    <title value="Knowledge Artifact Release"/>
+    <status value="draft"/>
+    <kind value="operation"/>
+    <date value="2023-01-27T09:29:23+11:00"/>
+    <publisher value="HL7 (FHIR Project)"/>
+    <contact>
+        <telecom>
+            <system value="url"/>
+            <value value="http://hl7.org/fhir"/>
+        </telecom>
+        <telecom>
+            <system value="email"/>
+            <value value="fhir@lists.hl7.org"/>
+        </telecom>
+    </contact>
+    <description value="The release operation supports updating the status of an existing draft artifact to active. The operation sets the date element of the resource and pins versions of all direct and transitive references. Child artifacts (i.e. artifacts that compose the existing artifact) are also released, recursively."/>    
+    <code value="knowledge-artifact-release"/>
+    <comment value="The release operation supports the ability of a repository to transitively gather components and dependencies of the artifact and record those as dependencies on the artifact itself while updating the status of all components to 'active'. For both components and dependencies if versions are not specified in the relevant reference, the operation will infer the version to be used by either locating a current 'draft' version of the artifact being reference or locating the latest active version in the case of dependencies."/>
+    <resource value="Resource"/>
```

Comment:
Add that the version determined should be recorded in the "manifest"

Reply:
Description and comment have both been updated significantly - beyond just the requested mention of recording references in the manifest. Please review the comment value closely.